INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 135
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:60 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:60 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:60 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:60 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:60 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:60 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.113 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.113 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 10563 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:49397
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "GET /session/6cacc9822701fc9793079e6f98109d3d/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/element/f.FBF48944F955AAAC5929162732648B21.d.BB43971F06871DD7955B2C1EC496A4B9.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/element/f.FBF48944F955AAAC5929162732648B21.d.BB43971F06871DD7955B2C1EC496A4B9.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/element/f.FBF48944F955AAAC5929162732648B21.d.BB43971F06871DD7955B2C1EC496A4B9.e.9/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "GET /session/6cacc9822701fc9793079e6f98109d3d/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "GET /session/6cacc9822701fc9793079e6f98109d3d/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 153
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: 5d53ea4482eef1f69024e2c3bdc6c0cfdb39ec2e21ae0b6462286e2e1f28ab9c
DEBUG    LoginPage:login_page.py:32 login crumb: 5d53ea4482eef1f69024e2c3bdc6c0cfdb39ec2e21ae0b6462286e2e1f28ab9c
DEBUG    MainPage:ui_element.py:59 Click on locator ('link text', 'Manage Jenkins')
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('link text', 'Manage Jenkins'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "GET /session/6cacc9822701fc9793079e6f98109d3d/element/f.FBF48944F955AAAC5929162732648B21.d.5332EE3D0F5DFCBCD2CAAA7E5416CED5.e.37/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/element/f.FBF48944F955AAAC5929162732648B21.d.5332EE3D0F5DFCBCD2CAAA7E5416CED5.e.37/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "GET /session/6cacc9822701fc9793079e6f98109d3d/url HTTP/1.1" 200 0
DEBUG    ManageJenkinsPage:ui_element.py:59 Click on locator ('xpath', "//a[@href='load-statistics']/..")
DEBUG    ManageJenkinsPage:ui_element.py:28 waiting for element_to_be_clickable(('xpath', "//a[@href='load-statistics']/.."),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "GET /session/6cacc9822701fc9793079e6f98109d3d/element/f.FBF48944F955AAAC5929162732648B21.d.6A235BC9F2BE2D45907C815D1DB1FEC1.e.76/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "POST /session/6cacc9822701fc9793079e6f98109d3d/element/f.FBF48944F955AAAC5929162732648B21.d.6A235BC9F2BE2D45907C815D1DB1FEC1.e.76/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "GET /session/6cacc9822701fc9793079e6f98109d3d/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "GET /session/6cacc9822701fc9793079e6f98109d3d/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:49397 "DELETE /session/6cacc9822701fc9793079e6f98109d3d HTTP/1.1" 200 0